docs/tails_printing_guide.rst

```diff
@@ -7,12 +7,43 @@ potentially untrusted network. This is why **we strongly recommend using
 a printer that does not have WiFi or Bluetooth**, and hooking up to it
 using a regular USB cable to print.
 
-Normally, any printer should work with Tails "out of the box." Most
-difficulties stem from not selecting the right driver (extra software
-needed for the printer and computer to communicate). Luckily, Tails has
-a large number of drivers for just about any popularly manufactured
-printer on hand, without even having to download new drivers from the
-web.
+Installing and Printing via the Tails GUI
+-----------------------------------------
+
+Normally, any printer should work with Tails "out of the box." Let's look at
```

Comment:
While you're making changes to the printer docs, you might as well resolve #1149 by updating this sentence to more accurately reflect reality. The core concern of #1149 is that the printing guide says "any printer should work with Tails out of the box," which conflicts with language elsewhere in the documentation and with our experience generally. I suggest revising this sentence to something like:

> Finding a printer that works with Tails can be challenging because Tails is based on the Linux operating system, which often has second-class hardware support in comparison to operating systems such as Windows or macOS. We maintain a list of printers that have personally tested and gotten to work with Tails in the Hardware guide [link to printers section of hardware guide]; if possible, we recommend using one of those printers. In any case, this document outlines the usual set of steps that we follow when attempting to use a new printer with Tails, and provides some troubleshooting tips that you may find useful if you are trying to use a different printer.